common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3060,6 +3060,12 @@
     ],
     "sqlState" : "22023"
   },
+  "MAP_KEY_BAD_DATATYPE" : {
```

Comment:
Will we support it eventually? If yes we can use a temp error class, with a name starting with `_LEGACY_ERROR_TEMP`

Reply:
I don't think that we plan to support Map with key of Map type, as it was forbidden by your PR earlier.